src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.cs

```diff
@@ -41,17 +41,15 @@ internal abstract partial class AbstractNavigateToSearchService : INavigateToSea
             };
         }
 
-        public async Task<NavigateToSearchLocation> SearchDocumentAsync(Document document, string searchPattern, IImmutableSet<string> kinds, Func<INavigateToSearchResult, Task> onResultFound, bool isFullyLoaded, CancellationToken cancellationToken)
+        public async Task SearchDocumentAsync(Document document, string searchPattern, IImmutableSet<string> kinds, bool isFullyLoaded, Func<INavigateToSearchResult, Task> onResultFound, CancellationToken cancellationToken)
```

Comment:
```suggestion
        public async Task SearchDocumentAsync(
        	Document document,
      		string searchPattern,
      		IImmutableSet<string> kinds,
      		bool isFullyLoaded,
      		 Func<INavigateToSearchResult, Task> onResultFound,
      		 CancellationToken cancellationToken)
```

Reply:
going to rollt his into the followup PR.